tests/query_workflow.go

```diff
@@ -175,27 +174,29 @@ func (s *ClientFunctionalSuite) TestQueryWorkflow_QueryWhileBackoff() {
 	s.NotNil(workflowRun)
 	s.True(workflowRun.GetRunID() != "")
 
-	// wait until retry with backoff is scheduled
-	findBackoffWorkflow := false
-	for i := 0; i < 5; i++ {
-		historyEvents := s.getHistory(s.namespace, &commonpb.WorkflowExecution{
-			WorkflowId: id,
-		})
-		if len(historyEvents) == 1 {
-			s.EqualHistoryEvents(`
-  1 WorkflowExecutionStarted {"FirstWorkflowTaskBackoff":{"Seconds":10}}`, historyEvents)
-			findBackoffWorkflow = true
-			break
-		}
+	_, err = s.sdkClient.QueryWorkflow(ctx, id, "", testname)
 
-		// wait for the retry, which will have backoff
-		time.Sleep(time.Second)
+	if expectError {
+		s.Error(err)
+		s.ErrorContains(err, consts.ErrWorkflowTaskNotScheduled.Error())
```

Comment:
Don't refer `const`. Hard code string here. It is a part of contract and should be checked explicitly.

Reply:
discussed offline. will continue as is.